Engine/source/gfx/bitmap/ddsLoader.cpp

```diff
@@ -607,7 +607,7 @@ bool DDSFile::read(Stream &s, U32 dropMipCount)
       else if ( mFlags.test( PitchSizeFlag ) )
          mPitchOrLinearSize = getSurfacePitch( dropMipCount );
       else
-         mPitchOrLinearSize = mPitchOrLinearSize; // Do nothing?
+         ; // Do nothing?
```

Comment:
If it's really a NOOP why not remove the `else` entirely? Also, I can't see what type `mPitchOrLinearSize` is, but maybe check for overloaded assignment?